streampipes-extensions/streampipes-processors-geo-jvm/src/main/java/org/apache/streampipes/processors/geo/jvm/jts/processor/trajectory/TrajectoryFromPointsProcessor.java

```diff
@@ -51,8 +51,9 @@ public class TrajectoryFromPointsProcessor extends StreamPipesDataProcessor {
   private static final String SUBPOINTS_KEY = "subpoints-key";
   private static final String DESCRIPTION_KEY = "description-key";
   private static final String TRAJECTORY_KEY = "trajectory-key";
-  private static final String TRAJECTORY_RUNTIME = "trajectoryWKT";
-  private static final String DESCRIPTION_RUNTIME = "trajectoryDescription";
+  private static final String TRAJECTORY_GEOMETRY_RUNTIME = "trajectory-geometry";
+  private static final String TRAJECTORY_EPSG_RUNTIME = "trajectory-epsg";
+  private static final String DESCRIPTION_RUNTIME = "trajectory-description";
```

Comment:
Do you not want to provide labels via the `strings.en` file?

Reply:
@bossenti 
so far I never implemented the runtime names to the string.en file
What would be the benefit of it?


All necessary descriptions for the user are already written down:
![traj_elements](https://user-images.githubusercontent.com/16739072/218331455-295f6d5d-f16c-4d64-9ca4-8fefc68f23d5.png)

Reply:
Ah sorry, my bad 🙈
Messed up the variables